concourse/scripts/pipelines-cloudfoundry.sh

```diff
@@ -11,9 +11,11 @@ config_autodelete="${SCRIPT_DIR}/../pipelines/autodelete-cloudfoundry.yml"
 
 generate_vars_file() {
    set -u # Treat unset variables as an error when substituting
+   local aws_account=${AWS_ACCOUNT:-dev}
    cat <<EOF
 ---
-aws_account: ${AWS_ACCOUNT:-dev}
+aws_account: ${aws_account}
+auto_deploy: $([ "${aws_account}" != "dev" ] && echo "true" || echo "false")
```

Comment:
Agree with all of that :+1: 

We agreed that we'd change this to a new environment variable name and raise a new story to create `Makefile` targets to group these together.

@Jonty can you check that [#114080971](https://www.pivotaltracker.com/story/show/114080971) makes sense?